doc/script_commands.txt

```diff
@@ -8288,6 +8288,9 @@ The command returns 0 upon success, and these values upon failure:
 
 Put -1 for x and y if want to warp player with default entrance coordinates.
 
+Note:
+Instance name is a legacy parameter and will be ignored internally.
```

Comment:
Woops you're right. Maybe I'll clean this up in a follow up too.
That could be done by `instance_searchtype_db(instance_data[instance_id]->type)` so these parameters really could be ignored for real in the future.